<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrinterPage</class>
 <widget class="QWidget" name="PrinterPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1673</width>
    <height>957</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Sans Serif</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <widget class="QFrame" name="pageFrame">
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="1" column="0">
       <widget class="SlidingStackedWidget" name="stackWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <widget class="QWidget" name="overviewWidget">
         <layout class="QGridLayout" name="gridLayout_18">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QScrollArea" name="scrollArea">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="pageContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>1669</width>
               <height>916</height>
              </rect>
             </property>
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="0" column="0">
               <widget class="QAnimatedToolBox" name="toolBox" native="true">
                <layout class="QGridLayout" name="gridLayout_19"/>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QWidget" name="sideWidget" native="true">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>460</width>
                  <height>16777215</height>
                 </size>
                </property>
                <layout class="QGridLayout" name="gridLayout_13">
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>9</number>
                 </property>
                 <property name="rightMargin">
                  <number>9</number>
                 </property>
                 <property name="bottomMargin">
                  <number>40</number>
                 </property>
                 <item row="0" column="0" colspan="2">
                  <widget class="QFrame" name="webcamGroupBox">
                   <property name="minimumSize">
                    <size>
                     <width>420</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <layout class="QGridLayout" name="webcamLayout">
                    <property name="leftMargin">
                     <number>0</number>
                    </property>
                    <property name="topMargin">
                     <number>0</number>
                    </property>
                    <property name="rightMargin">
                     <number>0</number>
                    </property>
                    <property name="bottomMargin">
                     <number>0</number>
                    </property>
                    <property name="spacing">
                     <number>0</number>
                    </property>
                    <item row="1" column="0">
                     <widget class="QWidget" name="webcamContentWidget" native="true">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="minimumSize">
                       <size>
                        <width>360</width>
                        <height>191</height>
                       </size>
                      </property>
                      <layout class="QGridLayout" name="gridLayout_35">
                       <property name="leftMargin">
                        <number>4</number>
                       </property>
                       <property name="topMargin">
                        <number>4</number>
                       </property>
                       <property name="rightMargin">
                        <number>4</number>
                       </property>
                       <property name="bottomMargin">
                        <number>4</number>
                       </property>
                      </layout>
                     </widget>
                    </item>
                    <item row="0" column="0">
                     <widget class="QWidget" name="webcamTitleBar" native="true">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="minimumSize">
                       <size>
                        <width>0</width>
                        <height>40</height>
                       </size>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>16777215</width>
                        <height>40</height>
                       </size>
                      </property>
                      <layout class="QGridLayout" name="gridLayout_33">
                       <property name="leftMargin">
                        <number>6</number>
                       </property>
                       <property name="topMargin">
                        <number>4</number>
                       </property>
                       <property name="rightMargin">
                        <number>4</number>
                       </property>
                       <property name="bottomMargin">
                        <number>4</number>
                       </property>
                       <item row="0" column="0">
                        <widget class="QLabel" name="webcamIconLabel">
                         <property name="minimumSize">
                          <size>
                           <width>32</width>
                           <height>32</height>
                          </size>
                         </property>
                         <property name="maximumSize">
                          <size>
                           <width>32</width>
                           <height>32</height>
                          </size>
                         </property>
                         <property name="text">
                          <string/>
                         </property>
                        </widget>
                       </item>
                       <item row="0" column="1">
                        <widget class="QLabel" name="label_7">
                         <property name="text">
                          <string>Webcam</string>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item row="1" column="0" colspan="2">
                  <widget class="QWidget" name="printJobWidget" native="true">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>420</width>
                     <height>100</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <layout class="QGridLayout" name="gridLayout_23">
                    <property name="leftMargin">
                     <number>0</number>
                    </property>
                    <property name="topMargin">
                     <number>0</number>
                    </property>
                    <property name="rightMargin">
                     <number>0</number>
                    </property>
                    <property name="bottomMargin">
                     <number>0</number>
                    </property>
                    <property name="spacing">
                     <number>0</number>
                    </property>
                    <item row="1" column="0" colspan="2">
                     <widget class="QWidget" name="printerStatusContents" native="true">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="minimumSize">
                       <size>
                        <width>0</width>
                        <height>128</height>
                       </size>
                      </property>
                      <layout class="QGridLayout" name="gridLayout_9"/>
                     </widget>
                    </item>
                    <item row="0" column="0" colspan="2">
                     <widget class="QWidget" name="printStatusTitleBar" native="true">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="maximumSize">
                       <size>
                        <width>16777215</width>
                        <height>50</height>
                       </size>
                      </property>
                      <layout class="QGridLayout" name="gridLayout_26">
                       <property name="leftMargin">
                        <number>6</number>
                       </property>
                       <property name="topMargin">
                        <number>4</number>
                       </property>
                       <property name="rightMargin">
                        <number>4</number>
                       </property>
                       <property name="bottomMargin">
                        <number>4</number>
                       </property>
                       <item row="0" column="1">
                        <widget class="QLabel" name="printerStatusTitleLabel">
                         <property name="text">
                          <string>Printer Status</string>
                         </property>
                        </widget>
                       </item>
                       <item row="0" column="0">
                        <widget class="QLabel" name="statusIconLabel">
                         <property name="minimumSize">
                          <size>
                           <width>32</width>
                           <height>32</height>
                          </size>
                         </property>
                         <property name="maximumSize">
                          <size>
                           <width>32</width>
                           <height>32</height>
                          </size>
                         </property>
                         <property name="text">
                          <string/>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item row="2" column="0" colspan="2">
                  <widget class="QFrame" name="fanFrame">
                   <property name="minimumSize">
                    <size>
                     <width>420</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="frameShape">
                    <enum>QFrame::Shape::StyledPanel</enum>
                   </property>
                   <property name="frameShadow">
                    <enum>QFrame::Shadow::Raised</enum>
                   </property>
                   <layout class="QGridLayout" name="fanFrameLayout">
                    <property name="leftMargin">
                     <number>0</number>
                    </property>
                    <property name="topMargin">
                     <number>0</number>
                    </property>
                    <property name="rightMargin">
                     <number>0</number>
                    </property>
                    <property name="bottomMargin">
                     <number>0</number>
                    </property>
                    <property name="spacing">
                     <number>0</number>
                    </property>
                   </layout>
                  </widget>
                 </item>
                 <item row="3" column="0" colspan="2">
                  <widget class="QWidget" name="widget_6" native="true">
                   <layout class="QGridLayout" name="gridLayout_14">
                    <item row="0" column="3">
                     <widget class="QToolButton" name="terminalButton">
                      <property name="minimumSize">
                       <size>
                        <width>100</width>
                        <height>100</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>...</string>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="4">
                     <widget class="QToolButton" name="settingsButton">
                      <property name="minimumSize">
                       <size>
                        <width>100</width>
                        <height>100</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>...</string>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="2">
                     <widget class="QToolButton" name="gcodeButton">
                      <property name="minimumSize">
                       <size>
                        <width>100</width>
                        <height>100</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>...</string>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="1">
                     <widget class="QToolButton" name="browserButton">
                      <property name="minimumSize">
                       <size>
                        <width>100</width>
                        <height>100</height>
                       </size>
                      </property>
                      <property name="text">
                       <string/>
                      </property>
                      <property name="toolButtonStyle">
                       <enum>Qt::ToolButtonStyle::ToolButtonIconOnly</enum>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="0" colspan="5">
                     <spacer name="verticalSpacer_2">
                      <property name="orientation">
                       <enum>Qt::Orientation::Vertical</enum>
                      </property>
                      <property name="sizeType">
                       <enum>QSizePolicy::Policy::Expanding</enum>
                      </property>
                      <property name="sizeHint" stdset="0">
                       <size>
                        <width>20</width>
                        <height>0</height>
                       </size>
                      </property>
                     </spacer>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="bedmeshWidget">
         <layout class="QGridLayout" name="gridLayout_11">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </widget>
        <widget class="QWidget" name="settingsWidget">
         <layout class="QGridLayout" name="gridLayout_12">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QFrame" name="actionBar">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="goBackButton">
           <property name="minimumSize">
            <size>
             <width>35</width>
             <height>35</height>
            </size>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SlidingStackedWidget</class>
   <extends>QStackedWidget</extends>
   <header>3rdparty/SlidingStackedWidget/slidingstackedwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QAnimatedToolBox</class>
   <extends>QWidget</extends>
   <header>ui/QAnimatedToolBox/qanimatedtoolbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
